app/src/interfaces/AlertHandlerInterface.ts

```diff
@@ -0,0 +1,5 @@
+import MetagameEventEvent from '../handlers/census/events/MetagameEventEvent';
+
+export default interface AlertHandlerInterface {
+    handleMetagameEvent(mgeEvent: MetagameEventEvent): boolean;
+}
```

Comment:
Just curious what purpose the alert interface will serve? Expansion for other event types maybe?

Reply:
I thought e.g. the GainXPHandler will need to know if the received event is happening in an alert and then determine if the event has to be stored or dropped.